robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -102,6 +160,10 @@ public void setLedMode(LedMode mode) {
         }
     }
 
+    /*
+     * Can control whether you want the Limelight to take snapshots or not. When it's taking snapshots, it will do it
```

Comment:
Start with `Sets` instead of `Can control`

Reply:
doen